Cargo.toml

```diff
@@ -3,20 +3,22 @@ authors = ["Antoni Boucher <bouanto@zoho.com>"]
 description = "Rust bindings for the VTE 3 library"
 license = "MIT"
 name = "vte-rs"
-version = "0.1.6"
+version = "0.2.0"
 
 [dependencies]
-bitflags = "^0.8.2"
-gdk = "^0.5.2"
-gio-sys = "^0.3.3"
-glib = "^0.1.2"
-glib-sys = "^0.3.3"
-gobject-sys = "^0.3.3"
-gtk = "^0.1.2"
-gtk-sys = "^0.3.3"
-libc = "^0.2.21"
-pango = "^0.1.2"
-vte-sys = "^0.2.0"
+bitflags = "1"
+gdk-sys = "0.8"
+gdk = "0.10"
+gio-sys = "0.8"
+gio = "0.6"
+glib = "0.7"
+glib-sys = "0.8"
+gobject-sys = "0.8"
+gtk = "0.6"
+gtk-sys = "0.8"
+libc = "0.2"
+pango = "0.6"
+vte-sys = { version="0.2.1", path="../vte-sys-rs"}
```

Comment:
Not sure if `..` in path is accepted by `cargo publish`.
It should probably be `./vte-sys-rs`.

Reply:
Ah, sorry, did that for local testing.

Reply:
I've added vte-sys-rs as submodule, or are you going to make a new release of vte-sys-rs?

Reply:
Only if it's needed.